sm_project/cl_app/views.py

```diff
@@ -31,6 +31,26 @@ def add_checklist(request):
     else:
         return redirect('user_app:login')
 
+"""Allows the user to remove themselves from a checklist's list of users.
```

Comment:
Hey @Lilys-Code, normally the docstring goes inside the relevant functino, this seems to be on top?

Reply:
otherwise looks good :-)